app/ante/fee_test.go

```diff
@@ -41,76 +41,109 @@ func TestValidateTxFee(t *testing.T) {
-			isCheckTx: false,
-			expErr:    false,
+			name:        "good tx; minFee = 0.8, rounds up to 1",
+			fee:         sdk.NewCoins(sdk.NewInt64Coin(appconsts.BondDenom, feeAmount)),
+			gasLimit:    400,
+			isCheckTx:   false,
+			expErr:      false,
+			minGasPrice: validatorMinGasPriceCoin,
 		},
 		{
-			name:      "good tx; gas limit and fee are maximum values",
-			fee:       sdk.NewCoins(sdk.NewInt64Coin(appconsts.BondDenom, math.MaxInt64)),
-			gasLimit:  math.MaxUint64,
-			isCheckTx: false,
-			expErr:    false,
+			name:        "good tx; fee above node's required minimum",
+			fee:         sdk.NewCoins(sdk.NewInt64Coin(appconsts.BondDenom, feeAmount+1)),
+			gasLimit:    uint64(float64(feeAmount) / validatorMinGasPrice),
+			isCheckTx:   true,
+			expErr:      false,
+			minGasPrice: validatorMinGasPriceCoin,
 		},
 		{
-			name:      "bad tx; gas limit and fee are 0",
-			fee:       sdk.NewCoins(sdk.NewInt64Coin(appconsts.BondDenom, 0)),
-			gasLimit:  0,
-			isCheckTx: false,
-			expErr:    false,
+			name:        "bad tx; fee below node's required minimum",
+			fee:         sdk.NewCoins(sdk.NewInt64Coin(appconsts.BondDenom, feeAmount-1)),
+			gasLimit:    uint64(float64(feeAmount) / validatorMinGasPrice),
+			isCheckTx:   true,
+			expErr:      true,
+			minGasPrice: validatorMinGasPriceCoin,
 		},
 		{
-			name:      "good tx; minFee = 0.8, rounds up to 1",
-			fee:       sdk.NewCoins(sdk.NewInt64Coin(appconsts.BondDenom, feeAmount)),
-			gasLimit:  400,
-			isCheckTx: false,
-			expErr:    false,
+			name:        "min gas price is empty",
+			fee:         sdk.NewCoins(sdk.NewInt64Coin(appconsts.BondDenom, feeAmount)),
+			gasLimit:    uint64(float64(feeAmount) / appconsts.DefaultMinGasPrice),
+			isCheckTx:   true,
+			expErr:      false,
+			minGasPrice: "", // should use the default min gas price
 		},
 		{
-			name:      "good tx; fee above node's required minimum",
-			fee:       sdk.NewCoins(sdk.NewInt64Coin(appconsts.BondDenom, feeAmount+1)),
-			gasLimit:  uint64(float64(feeAmount) / validatorMinGasPrice),
-			isCheckTx: true,
-			expErr:    false,
+			name:        "min gas price is empty",
+			fee:         sdk.NewCoins(sdk.NewInt64Coin(appconsts.BondDenom, feeAmount-1)),
+			gasLimit:    uint64(float64(feeAmount) / appconsts.DefaultMinGasPrice),
+			isCheckTx:   true,
+			expErr:      true,
+			minGasPrice: "", // should use the default min gas price
 		},
 		{
-			name:      "bad tx; fee below node's required minimum",
-			fee:       sdk.NewCoins(sdk.NewInt64Coin(appconsts.BondDenom, feeAmount-1)),
-			gasLimit:  uint64(float64(feeAmount) / validatorMinGasPrice),
-			isCheckTx: true,
-			expErr:    true,
+			name:        "min gas price is 0utia",
+			fee:         sdk.NewCoins(sdk.NewInt64Coin(appconsts.BondDenom, feeAmount)),
+			gasLimit:    uint64(float64(feeAmount) / appconsts.DefaultMinGasPrice),
+			isCheckTx:   true,
+			expErr:      false,
+			minGasPrice: "0utia", // should use the default min gas price
```

Comment:
I thought 0utia should still be usable if specified in config and should _not_ use default min gas price

```suggestion
			minGasPrice: "0utia", 
```